### PR TITLE
es2015ify

### PR DESCRIPTION
**Pull request by SamVerschueren**
- 💄 
- es2015ify
- :arrow_up: w3counter

### CODE REVIEW

**Comment by SamVerschueren** [comment]
// @sindresorhus

File: readme.md [hunk 1]
```diff
@@ -32,7 +32,7 @@ Gets ten most popular screen resolutions from [w3counter](http://www.w3counter.c
 ## CLI
 
 ```
-$ npm install --global get-res
+$ npm install -g get-res
```
Comment (sindresorhus) [inline readme.md]: should stay `--global`

**Comment by sindresorhus** [comment]
LGTM

**Comment by sindresorhus** [comment]
@kevva Whenever you have a chance, can you extract the CLI, and do a new major release?

**Comment by sindresorhus** [comment]
Thanks Sam :)

**Comment by SamVerschueren** [comment]
No problem, thanks for reviewing. If this one gets released, I will upgrade `pageres` :p.

**Comment by SamVerschueren** [comment]
ping @kevva (when you find some time :))

**Comment by SamVerschueren** [comment]
Noticed some new activity yesterday. Care to release this?

**Comment by kevva** [comment]
@SamVerschueren :D, yesss!

**Comment by SamVerschueren** [comment]
Awesome, thanks man!

**Comment by kevva** [comment]
Not a second too early :).

**Pull request merged**